### PR TITLE
enhance: option to make enter send message

### LINKED ISSUE #8952: Swap enter/ctrl+enter functionality for chat
## Summary

Like most other chat apps, it'd make sense if enter was send and ctrl+enter was new line.

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
# What
Like most other chat services, enter will send the message while ctrl/shift/meta enter will make a new line.

# Why
Close #8952 

# Additional info (optional)
Tested and working on https://stop.voring.me

### CODE REVIEW

**Comment by ThatOneCalculator** [comment]
Requesting review from @syuilo , everything works as expected

**Comment by ThatOneCalculator** [comment]
Drafting, as this should probably be an opt-in setting.

**Comment by ThatOneCalculator** [comment]
@syuilo @tamaina I made it optional and off by default

**Comment by ThatOneCalculator** [comment]
(Apologies for the machine translated Japanese for the locale string, I'm guessing it's not good)

**Comment by ThatOneCalculator** [comment]
Tested and working on my instance.

File: packages/client/src/pages/messaging/messaging-room.form.vue [hunk 1]
```diff
@@ -125,8 +125,21 @@ function onDrop(ev: DragEvent): void {
 
 function onKeydown(ev: KeyboardEvent) {
 	typing();
-	if ((ev.key === 'Enter') && (ev.ctrlKey || ev.metaKey) && canSend) {
-		send();
+	let sendOnEnter = localStorage.getItem('enterSendsMessage') === 'true' || defaultStore.state.enterSendsMessage;
```
Comment (syuilo) [inline packages/client/src/pages/messaging/messaging-room.form.vue]: Why is localStorage referenced?

Comment (ThatOneCalculator) [reply packages/client/src/pages/messaging/messaging-room.form.vue]: Would `computed(defaultStore.makeGetterSetter('enterSendsMessage'))` be better?

Comment (tamaina) [reply packages/client/src/pages/messaging/messaging-room.form.vue]: 単純にdefaultStore.state.enterSendsMessageで判定するべき

**Comment by tamaina** [comment]
p1.a9z.devで動作中

**Pull request closed without merging**